### PR TITLE
various upgrades

### PR DESCRIPTION
**Pull request by raffaelespazzoli**
Signed-off-by: raffaelespazzoli <raffaele.spazzoli@gmail.com>

### CODE REVIEW

File: integration/kube-prometheus-stack-values.yaml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    ## The test-metrics sidecar will become ready when metrics are available.
+    containers: 
+      - image: busybox:latest
+        name: test-metrics
+        command: 
+        - /bin/sh
+        - -c
+        - | 
+            while true; do
+              out=$(wget -O - --post-data="query=controller_runtime_active_workers%7Bnamespace%3D%22volume-expander-operator-local%22%7D%0A" localhost:9090/api/v1/query)
```
Comment (sabre1041) [inline integration/kube-prometheus-stack-values.yaml:54]:
```suggestion
              out=$(wget -O - --post-data="query=controller_runtime_active_workers%7Bnamespace%3D%22keepalived-operator-local%22%7D%0A" localhost:9090/api/v1/query)
```

**Review by sabre1041 — CHANGES_REQUESTED** [review]
@raffaelespazzoli this may be the reason for the failed CI runs

File: integration/kube-prometheus-stack-values.yaml (hunk 1 above)
Comment (raffaelespazzoli) [reply integration/kube-prometheus-stack-values.yaml]: done

**Review by sabre1041 — APPROVED** [review]
LGTM

**Pull request merged**